src/@types/jest.d.ts

```diff
@@ -0,0 +1,13 @@
+namespace jest {
+  import { QueryParamConfig } from 'serialize-query-params'
```

Comment:
Still need to verify why importing external type definitions inside `d.ts` is not working and being set to `any` - Will handle that in a next PR since this one is taking too long